cpu/cpustress.py

```diff
@@ -65,12 +65,12 @@ def setUp(self):
         """
         Check required packages is installed, and get current SMT value.
         """
+        if 'ppc' not in platform.processor():
```

Comment:
As it is already being moved, could you also use the avocado library `distro.detect().arch` for this purpose?

Reply:
Done @harish-24